tests/drivers/dma/chan_blen_transfer/boards/lpcxpresso55s36.overlay

```diff
@@ -0,0 +1 @@
+test_dma0: &dma0 { };
```

Comment:
should this board be added to the test yaml?

Reply:
the testcase.yaml is setup for all dma supported devices on the chan blen test

Reply:
okay